quadlets/audiobookshelf/audiobookshelf.container

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Audiobookshelf Quadlet
+
+[Service]
+Restart=always
+TimeoutStartSec=900
+
+[Install]
+WantedBy=default.target
+
+[Container]
+Image=docker.io/advplyr/audiobookshelf:latest
+ContainerName=audiobookshelf
+HostName=audiobookshelf
+
+PublishPort=13378:80
```

Comment:
I happen to know this one is an upstream suggestion, but I wonder if we should just stick to a simple `80:80` mapping. I intend to demonstrate advanced concepts with documentation, and in `pods` once I finish working out the `mamstack`.

Do you think it's more or less confusing if we set this to `80:80`?

Reply:
This would be a problem for anyone running a web server as 80 is usually open and redirects to 443 this is why I redirect it as my host is using 80 and then a 301 to 443. This is also a problem if you run adguard.

Reply:
I'll have to think on this one a lot more, because the upstream example also uses this.